### ISSUE TITLE
Dataset availbility

### ISSUE THREAD

**Issue by philschmid** [DaoD/INTERS#3]
Hey, are you planning to share the dataset?

**Comment by ilml** [comment]
+1 to this

**Comment by DaoD** [comment]
Yup! We will share the fine-tuned models, datasets, templates, and codes within a month. 

Thanks!

**Comment by CAholder** [comment]
> Yup! We will share the fine-tuned models, datasets, templates, and codes within a month.
> 
> Thanks!

Very much looking forward to it! This was a fascinating paper

**Comment by CAholder** [comment]
+1 hopefully the information drops soon!

**Comment by DaoD** [comment]
Dear all, 

Thanks for your interest in our study. 

We have just released the training set, part of the test set, and instruction templates. Other resources are still in preparation (due to the network latency, it may take several days to upload the model).

Regards,
Z.
